packages/shared/styles/helpers/_layout.scss

```diff
@@ -12,3 +12,19 @@
   border: var(#{$element}, _border($element, $border-style, $border-color));
   border-width: var(#{$element}-width, $border-width);
 }
+
+// Use this mixin to set component background properties: color, image, position, size
+@mixin background(
+  $element,
+  $color,
+  $image: url(""),
+  $position: 0% 0%,
+  $size: cover
+) {
+  background: var(
+    #{$element},
+    var(#{$element}-image, $image) var(#{$element}-color, $color) no-repeat
```

Comment:
@pspaczek wdyt to add also a variable for the repeat property, and set no-repeat as a default? :)

Reply:
added, but someone uses repeat? :D